modules/carto/src/api/parseMap.ts

```diff
@@ -217,7 +217,7 @@ function createChannelProps(
     );
   }
 
-  if (heightField) {
```

Comment:
It's not related with raster, right?

Reply:
Rasters do support extrusion (comes with perf hit), but yes this was an issue I just found while developing that is unrelated to this PR (fetchMap doesn't support raster)